lib/LedgerSMB/Scripts/import_csv.pm

```diff
@@ -114,7 +114,7 @@ sub _aa_multi {
         $acst->execute($acct);
         ($pass) = $acst->fetchrow_array;
         $request->error("Account $acct not found") if !$pass;
-        ($acct) = split /--/, $ref->[3];
+        ($acct) = split /--/, $ref->[3];  ## no critic (ProhibitMagicNumbers) sniff
         $acst->execute($acct);
```

Comment:
The columns in the CSV input aren't accessed by heading, but by number. Maybe we should create constants which map the numbered columns to mnemonics?

Reply:
I suspect that when I see a batch being run this may make sense, but static analysis is losing in a cost/benefit way.